cloud/maintenance.md

```diff
@@ -67,4 +67,47 @@ downtime is usually between 30&nbsp;seconds and 5&nbsp;minutes. We endeavor to
 notify you on email ahead of the upgrade if downtime is required, so that you
 can plan accordingly. However, in some cases, we might not be able to do so.
 
+## Upgrade to a new PostgreSQL version
+Timescale currently supports PostgreSQL&nbsp;12 and 13. You can manually upgrade
+to the next PostgreSQL version in the Timescale Cloud dashboard.
+
+Upgrading to a newer version of PostgreSQL allows you to take advantage of new
+features, enhancements, and security fixes. It also ensures that you are using a
+version of PostgreSQL that is compatible with the newest version of TimescaleDB,
+allowing you to take advantage of everything Timescale has to offer. For more
+information about feature changes between versions, see the
+[PostgreSQL release notes][postgres-relnotes] and
+[TimescaleDB release notes][timescale-relnotes].
+
+<highlight type="warning">
+Your Timescale Cloud service is unavailable for use until the upgrade is
+complete. Upgrading can take up to several hours to complete, so we recommend
+that you plan ahead, and upgrade during a time with low usage.
+</highlight>
+
+If you want to try out the upgrade before you run it on your production system,
+you can create a fork of your service, and attempt the upgrade on the fork
+before you start. This can give you a good idea of what happens during an
+upgrade, and how long it might take in your use case. For more infomration about
+creating a fork of your service, see the [Operations
+section][operations-forking].
```

Comment:
Before you upgrade your service in the production system, you can create a fork of the service, and upgrade  the fork.  This helps you understand what happens during an upgrade and estimate how long it might take for the upgrade. For more information about creating.....

Reply:
You can make suggestions directly by clicking the `Add a suggestion` button: 
![image](https://user-images.githubusercontent.com/3914967/153518075-1744b12d-f39a-4f4a-8ebd-c3a73c50399b.png)